### PR TITLE
[DR-3413] Remove de-elasticsearch chart

### PR DESCRIPTION
**Pull request by samanehsan**
Ticket: https://broadworkbench.atlassian.net/browse/DR-3413

Elasticsearch is configured for TDR and is currently enabled in dev, but it is not being used (it was added as a proof-of-concept). 

Note: This PR can only be merged after removing the elasticsearch override in dev: https://github.com/broadinstitute/terra-helmfile/pull/4906

### CODE REVIEW

File: charts/datarepo/Chart.yaml [hunk 1]
```diff
@@ -15,6 +15,8 @@ type: application
 
 version: 0.1.815
 appVersion: 0.1.815
+version: 0.2.0
+appVersion: 0.2.0
```
Comment (pshapiro4broad) [inline charts/datarepo/Chart.yaml:19]: Is this correct? I'd expect the new version / appVersion to replace the old one
```suggestion
version: 0.2.0
appVersion: 0.2.0
```

**Review by pshapiro4broad — APPROVED** [review]
One minor note, looks OK otherwise

File: charts/datarepo/Chart.yaml (hunk 1 above)
Comment (samanehsan) [reply charts/datarepo/Chart.yaml:19]: Ah that's my mistake from resolving merge conficts 🤦‍♀️

File: charts/datarepo/Chart.yaml [hunk 2]
```diff
@@ -13,8 +13,8 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version
 
-version: 0.1.815
-appVersion: 0.1.815
+version: 0.2.0
```
Comment (nmalfroy) [inline charts/datarepo/Chart.yaml:16]: Good call on the version rev

**Review by nmalfroy — APPROVED** [review]

**Pull request merged**